modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,76 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-install-logging-addon_{context}"]
+= Install logging add-on services on {product-title}
+
+{product-tite} (ROSA) does not come preconfigured with a logging solution, but logging can be enabled by manually installing the `cluster-logging-operator` service.
+
+Log forwarding capability is provided through the Amazon CloudWatch add-on service.
+
+.Prerequisites
+
+Request the following link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[quotas from AWS]:
+* Quota for the `cluster-logging-operator` add-on
+* Quota for the Amazon CloudWatch add-on
+
+.Procedure
+
+To install log forwarding using interactive mode:
+
+. Enter the following command:
++
+[source,terminal]
+----
+$ rosa install addon cluster-logging-operator --cluster=<cluster_name> --interactive
+----
++
+For `<cluster_name>`, enter the name of your cluster.
+
+. When prompted, enter `yes` to install `cluster-logging-operator`.
+. When prompted, enter `yes` to install the optional Amazon CloudWatch log forwarding add-on.
+. Enter `yes` or `no` for the log type prompts: applications, infrastructure, and audit logs.
++
+*Applications logs*
++
+Lets the Operator collect application logs, which includes everything that is *not* deployed in the `openshift-*` namespaces.
+
++
+*Infrastructure logs*
++
+Lets the Operator collect logs from OpenShift Container Platform, Kubernetes, and some nodes.
+
++
+*Audit logs*
++
+Lets the Operator collect node logs related to security audits.
+
+. If you are installing the Amazon CloudWatch add-on, enter the log grouping type: `logType`, `<namespaceName>``, or `<namespaceUUID>`.
```

Comment:
Again, just unsure about whether this is optional or not. If it's optional, then maybe the prereq has to mention the quota is optional

Reply:
It is optional, which I've now clarified in the introductory paragraph and in the pre-reqs. This sentence should make sense now, and I have clarified what that default is.